R/comment_linters.R

```diff
@@ -28,7 +28,7 @@ ops <- list(
 #' @export
 commented_code_linter <- function(source_file) {
   res <- re_matches(source_file$file_lines,
-                    rex("#", any_spaces,
```

Comment:
Maybe a better solution would be `some_of("#")`, which corresponds to `#+` and would allow any number of `#`

Reply:
I thought of it, but I was worried to disrupt something else for people running lintr in RStudio or other interfaces that I don't use. So I opted for the minimal possible code change that would fix my problem. But either of these options would totally work for me :)